src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -375,6 +381,26 @@ void ThrusterDynamicEffector::addThruster(THRSimConfig* newThruster, Message<SCS
 }
 
 
+/*! This method is used to update the blow down effects to the thrusters
+* at every computeForceTorque call when the thrusters are attached to a fuel
+* tank subject to blow down effects.
+ @return void
+ */
+void ThrusterDynamicEffector::ComputeBlowDownDecay()
+{
+    double currentFuelMass = this->fuelMass->getState()(0,0);
+    double massPercentage = currentFuelMass / this->maxFuelMass;
+
+    std::vector<THRSimConfig>::iterator it;
+    THROperation *ops;
```

Comment:
:small_red_triangle_down: You can delete this line and allow the compiler to infer the type of the variable within the loop using `auto`.